### PR TITLE
New version: CiderCollective.Cider.Nightly version 1.4.3.1847

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/58618)

### CODE REVIEW

File: manifests/c/CiderCollective/Cider/Nightly/1.4.3.1847/CiderCollective.Cider.Nightly.installer.yaml [hunk 1]
```diff
@@ -2,20 +2,20 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
 
 PackageIdentifier: CiderCollective.Cider.Nightly
-PackageVersion: 1.4.3.1827
+PackageVersion: 1.4.3.1847
```
Comment (ItzLevvie) [inline manifests/c/CiderCollective/Cider/Nightly/1.4.3.1847/CiderCollective.Cider.Nightly.installer.yaml:5]:
```suggestion
PackageVersion: 1.4.3-alpha.1847
```

File: manifests/c/CiderCollective/Cider/Nightly/1.4.3.1847/CiderCollective.Cider.Nightly.locale.en-US.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
 
 PackageIdentifier: CiderCollective.Cider.Nightly
-PackageVersion: 1.4.3.1827
+PackageVersion: 1.4.3.1847
```
Comment (ItzLevvie) [inline manifests/c/CiderCollective/Cider/Nightly/1.4.3.1847/CiderCollective.Cider.Nightly.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.4.3-alpha.1847
```

File: manifests/c/CiderCollective/Cider/Nightly/1.4.3.1847/CiderCollective.Cider.Nightly.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
 
 PackageIdentifier: CiderCollective.Cider.Nightly
-PackageVersion: 1.4.3.1827
+PackageVersion: 1.4.3.1847
```
Comment (ItzLevvie) [inline manifests/c/CiderCollective/Cider/Nightly/1.4.3.1847/CiderCollective.Cider.Nightly.yaml:5]:
```suggestion
PackageVersion: 1.4.3-alpha.1847
```

**Review by ItzLevvie — APPROVED** [review]

**Comment by vedantmgoyal2009** [comment]
@ItzLevvie please don't close the pull requests. They are marked as draft for purpose. The feature for automation (about which I had told you earlier) was being initially made using docker, but since it was slowing down the automation, I had to recreate it to run directly on the github runner. After my exams, I will get time to complete and do the testing and finally merge it into stable. The reason I'm using already open PRs is to ensure it updates the PR of the same package and doesn't overwrite the updates of one package onto the another package's PR. More number of PRs help in covering the edge cases.

Although, you can close them but then I will have manually create PRs for testing the logic and hence, I will increase my workload.

**Pull request merged**